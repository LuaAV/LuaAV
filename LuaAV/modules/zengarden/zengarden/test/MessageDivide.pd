#N canvas 764 231 450 300 10;
#X obj 21 19 loadbang;
#X msg 140 71 13;
#X obj 21 174 print;
#X msg 111 71 x;
#X msg 51 71 4;
#X msg 21 71 5 7;
#X msg 170 71 x;
#X msg 200 71 5;
#X obj 21 40 t b b b b b b b b b;
#X msg 233 71 2 3 4;
#X obj 21 151 / 5;
#X msg 280 71 5;
#X msg 80 71 0;
#X connect 0 0 8 0;
#X connect 1 0 10 0;
#X connect 3 0 10 0;
#X connect 4 0 10 0;
#X connect 5 0 10 0;
#X connect 6 0 10 1;
#X connect 7 0 10 0;
#X connect 8 0 5 0;
#X connect 8 1 4 0;
#X connect 8 2 12 0;
#X connect 8 3 3 0;
#X connect 8 4 1 0;
#X connect 8 5 6 0;
#X connect 8 6 7 0;
#X connect 8 7 9 0;
#X connect 8 8 11 0;
#X connect 9 0 10 0;
#X connect 10 0 2 0;
#X connect 11 0 10 0;
#X connect 12 0 10 1;
