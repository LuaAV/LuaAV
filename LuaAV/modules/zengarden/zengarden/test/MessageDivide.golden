[@ 0.000ms] print: 1
[@ 0.000ms] print: 0.666667
[@ 0.000ms] print: 1.66667
[@ 0.000ms] print: 4.33333
[@ 0.000ms] print: 0
[@ 0.000ms] print: 0.714286
